### PR TITLE
Adding meeting notes for 2018-11-26

### PR DESCRIPTION
**Pull request by vmx**

### CODE REVIEW

File: meeting-notes/2018/2018-11-26--ipld-sync.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+@mikeal There is no canonical format for IPLD nodes. Converting between JSON-LD and the CBOR equivalents is managed by the developers. --@mikeal did I get this right?
+
+@daviddias There is a lot of conversation about the IPLD DAG HTTP/RPC API https://github.com/ipfs/interface-ipfs-core/issues/81#issuecomment-277271941. DAG CBOR has less defined structure then the older Protbuf implementation, so there are some things to still work out.
+
+@volker When we talk about the HTTP API we assume it's a JSON API. Does it have to be JSON or can it be something else?
+ - @stebalien we can't currently just send raw data because we can't send metadata in headers (not supported by our HTTP/RPC API). JSON makes this easier though.
+ 
+@warpfork it would be nice to have more people dedicated to IPLD, especially since there is insufficient people in Go land to review @warpfork's code.
+ - @daviddias utilizing older Git issues and publishing specs will help get people involved in reviewing code.
```
Comment (daviddias) [inline meeting-notes/2018/2018-11-26--ipld-sync.md]: @warpfork check in and invite everyone from https://github.com/ipld/ipld/issues/56 and the previous IPLD deep dives https://github.com/ipfs/team-mgmt/issues/484

I'm sure multiple people will be delighted to learn that IPLD is becoming its full org with a fully dedicated team :)

//cc @nicola, @lgierth, @hermanjunge, @Kubuxu, @pgte, @wanderer, @kumavis, @jonchoi, @jonnycrunch, @b5, @mildred, @davidar, @davidad, @krl et al :D

File: meeting-notes/2018/2018-11-26--ipld-sync.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- **Attendees:**
+  - @aschmahmann
+  - @daviddias
+  - @mikeal
+  - @warpfork
+  - @mgoelzer
+  - @bigs
+  - @stebalien
+  - @vmx
+- **Recording:** https://www.youtube.com/watch?v=4ZxoQQnhmD4
```
Comment (daviddias) [inline meeting-notes/2018/2018-11-26--ipld-sync.md]: Thank you for uploading the recording. I created a playlist with all the IPLD Calls to make them easy to find:

Playlist url https://www.youtube.com/watch?v=103DT-im1_M&list=PLuhRWgmPaHtRfR6VhISbSiPJ87rsN3Pb9

File: meeting-notes/2018/2018-11-26--ipld-sync.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+@stebalien
+ Please comment on mentioned issue if you're interested. 
+ Some thoughts on moving CIDs to multiformats from IPLD, but likely doesn't make a huge difference.
+
+@warpfork IPLD interfaces not used very actively in Go projects yet.  Want project suited to be beta user / driver for go-ipld-prime development.  Maybe start go-unixfsv2 repo.
+
+
+@daviddias
+ - Why is unixfsv2 dragging? Why are so many people assigned?
```
[the remark this answers is not in the document]
Comment (Stebalien) [reply meeting-notes/2018/2018-11-26--ipld-sync.md]: I agree there needs to be precisely one owner. This clearly isn't working.

[the remark this answers is not in the document]
Comment (mikeal) [reply meeting-notes/2018/2018-11-26--ipld-sync.md]: What is the problem with having different people assigned to the spec and the implementation in each language? At the very least we should expect that the implementation in each language would have different assignees.

BTW, the draft spec has landed (not 100% finished but there are additional threads about those issues) and a standalone JS implementation has been updated to the latest spec.

[the remark this answers is not in the document]
Comment (Stebalien) [reply meeting-notes/2018/2018-11-26--ipld-sync.md]: The issue is that having multiple people ultimately responsible for the same task diffuses responsibility. Nobody knows who's actually responsible for what so everyone works on things they *know* they're *solely* responsible for.

Really, we should probably have "owner" and "assigned".

>  This clearly isn't working.

This was too harsh. You're right, progress *is* being made. At the end of the day, I think the correct assignment here is:

* Owner: @mikeal 
* Assigned: @warpfork, @vmx, @achingbrain.

[the remark this answers is not in the document]
Comment (vmx) [reply meeting-notes/2018/2018-11-26--ipld-sync.md]: I'm not involved that much as I didn't really see the need so far. I trust @mikeal @warpfork and @achingbrain enough that they'll make the right decisions :)

**Pull request merged**